python/tk_multi_workfiles/actions/new_task_action.py

```diff
@@ -51,4 +51,33 @@ def execute(self, parent_ui):
         res, new_task_form = app.engine.show_modal("Create New Task", app, NewTaskForm, self._entity, self._step,
                                                    g_user_cache.current_user, parent_ui)
 
-        return res == QtGui.QDialog.Accepted
+        if res != QtGui.QDialog.Accepted:
+            return False
+
+        try:
+            from sgtk.util.metrics import EventMetric
+
+            pipeline_step=new_task_form._get_pipeline_step()
+            task_name = pipeline_step.get("code","unknown") # Why not new_task_form._get_task_name() ?
+            entity_type = pipeline_step.get("type", "Unknown")
+            method = "Form" # since this was created from the Qt widget
+            fields_used = []
+            properties = {
+                "Linked Entity Type": entity_type,
```

Comment:
just do `"Linked Entity Type": pipeline_step.get("type", "Unknown")` and remove line 62.